MERN/Topic/Backend/Node/Synchronous and Asynchronous #2620.md

```diff
@@ -0,0 +1,53 @@
+# <p align=center> Synchronous and Asynchronous  </p>
+## <p align=center>Synchronous:-</p> 
+### &nbsp; &nbsp; Synchronous (or sync) execution usually refers to code executing in sequence. In sync programming, the program is executed line by line, one line at a time. Each time a function is called, the program execution waits until that function returns before continuing to the next line of code.
+ For Example :-  
+ ```javascript
+ <script>
+	document.write("Good Morning");   //First
+  document.write("<br>");
+	document.write("Good Afternoon") ;  //Second
+	document.write("<br>");
+	document.write("Good Night");  //Third
+</script>
+
+Output :- 
+Good Morning
+Good Afternoon
+Good Night
+```
+### In the above code snippet, the first line of the code "Good Morning " will be logged first then the second line " Good Afternoon " will be logged and then after its completion, the third line would be logged "Good Night ".So as we can see the codes work in a sequence. Every line of code waits for its previous one to get executed first and then it gets executed.**
+
+## <p align=center>Asynchronous:-</p> 
+###  &nbsp; &nbsp; Asynchronous (or async) execution refers to execution that doesn’t run in the sequence it appears in the code. In async programming the program doesn’t wait for the task to complete and can move on to the next task.
```

Comment:
Minor correction here. Don't just say that it doesn't run in the sequence. If it does not involves any heavy operations then even in async one sequence may follow.
Instead try to say something like `Its not mandatory`.